code/datums/commodity.dm

```diff
@@ -4,8 +4,8 @@
 	var/comtype = null // Type Path of the item on the market
 	var/price = 0 // Current selling price for this commodity
 	var/baseprice = 0 // Baseline selling price for this commodity
-	var/onmarket = 1 // Whether this item is currently being accepted for sale
-	var/indemand = 0 // Whether this item is currently being bought at a high price
+	var/onmarket = 0 // Whether this item is currently being accepted for sale on the shipping market
```

Comment:
Overall note: inverting this flag makes reviewing this PR a bit of a nightmare, as it's not clear what other stuff (that wasn't touched but has had its default changed by this parent change) has been altered.

Reply:
A LOT of stuff used the onmarket flag that shouldn't have. Basically, it was used in 0 places after it was defined, so people just kindof set whatever they wanted. It did nothing, before.